internal/driver/registry_default.go

```diff
@@ -334,6 +353,18 @@ func (r *RegistryDefault) WriteRouter() http.Handler {
 	return handler
 }
 
+func (r *RegistryDefault) MetricsRouter() http.Handler {
+	n := negroni.New(reqlog.NewMiddlewareFromLogger(r.Logger(), "keto").ExcludePaths(prometheus.MetricsPrometheusPath))
+	router := &x.MetricsRouter{Router: httprouter.New()}
+
+	r.PrometheusManager().RegisterRouter(router.Router)
+	r.MetricsHandler().SetRoutes(router.Router)
+	n.UseHandler(router)
+	n.Use(r.PrometheusManager())
+
```

Comment:
Here we are missing the CORS middleware, add it the same way as above:
```suggestion

	var handler http.Handler = n
	options, enabled := r.Config().CORS("metrics")
	if enabled {
		handler = cors.New(options).Handler(handler)
	}

```

Reply:
And you will have to add that interface to the CORS config provider.